components/atoms/image-carousel.html

```diff
@@ -112,32 +106,66 @@
+          </div>
+          <div id="carousel-image-2" class="carousel-image"></div>
+          <div id="carousel-image-3" class="carousel-image"></div>
       </div>
 
-    </div>
-    <div class="video-txt-wrapper">
+     
+
+      <div class="button-wrapper">
+         <span class="dots" id="0" onclick="slide(this)"></span>
+         <span class="dots" id="1" onclick="slide(this)"></span>
+         <span class="dots" id="2"onclick="slide(this)"></spna>
+       </div>
+     </div>
+
+
+    <div class="video-and-txt-wrapper">
       <div class="txt-wrapper">
           <h1>Refcode</h1>
              <p>Refcode is a curriculum to introduce beginners to coding. It is put on by the Refugee Career Hub as part of their computer literacy program. After going through this curriculum, students will learn the basics of how to build a website, and students will know a little bit about what working as a professional developer is like.</p>
              <p>Students will learn to use the tools that professional developers use to create websites. In this course, students will be introduced to Atom (an IDE or code editor), Chrome Inspect tools, GitHub Desktop, Cyberduck (an FTP client), and Freenom.com (a domain registrar).</p>
              <h2>Professional Skills</h2>
-                <p>In this class we will learn about the strategies, workflows, and tools that professionals use to stay organized, communicate, and manage their time. We will learn about Agile Methodology, stand-up meetings, sprint cycles, kanban boards (with Trello), Slack chat, GitHub, and pair-programming.
+             <p>In this class we will learn about the strategies, workflows, and tools that professionals use to stay organized, communicate, and manage their time. We will learn about Agile Methodology, stand-up meetings, sprint cycles, kanban boards (with Trello), Slack chat, GitHub, and pair-programming.
                 Each student will work towards creating a personal website by the end of the class. The student will register their own domain name and upload it to a web hosting server (provided by Refugee Career Hub).
-                
-                </p>
-            </div>
+             </p>
+      </div>
+
         <div class="video-wrapper">
             <iframe src="https://www.youtube.com/embed/Sw4r03j1Oc8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
         </div>
     </div>
     
   </markup>
+  <script>
+  let carousel_image_wrapper = document.getElementById('carousel-image-wrapper')
+  function slide(image){
+    carousel_image_wrapper.style.transform = `translateX(-${image.id}00%)`
+  }
+    setTimeout(function (){
+      // alert('hello')
+      carousel_image_wrapper.style.transform = `translateX(-100%)`
+
+    },2000)
+
+    setTimeout(function (){
+      // alert('hello')
+      carousel_image_wrapper.style.transform = `translateX(-200%)`
+
+    },12000)
+  
+  </script>
```

Comment:
You can save this script in a file called `image-carousel.js` (in the same folder as `image-carousel.html`).

Reply:
ok.